tests/pl/fan_HassTurnOn.yaml

```diff
@@ -5,6 +5,7 @@ tests:
       - "Uruchom w salonie wszystkie wentylatory"
       - "Włącz wentylatory w salonie"
       - "Uruchom wentylator w salonie"
+      - "Uruchom całe wentylatory w salonie"
```

Comment:
again here "całe wentylatory" is not good sentence, I only heard "wszystkie wentylatory"